Lib/pathlib.py

```diff
@@ -578,13 +542,20 @@ def parts(self):
             self._parts_tuple = tuple(self._parts)
             return self._parts_tuple
 
+    def makepath(self, *args):
```

Comment:
This makes a lot of sense as a concept: `__new__` requires much more thought during subclassing.

However, are we certain that unpacking is the best option for performance? Have you tried benchmarking it?

Reply:
Are you suggesting a signature like `makepath(self, args)`? It might be slightly faster but it's incongruous with the initialiser and `joinpath()`

Reply:
Oooh right... I compared it to `_make_child_relpath` instead but comparing to a public interface is more fair. I guess you're right, consistency takes precedence.